### DISCUSSION TITLE
Telegram alarm message can support markdown format?

### DISCUSSION THREAD

**Discussion by bbluo** [grafana/grafana#51004] [Alerting]
// I see the code only supports: parse_mode="html"
func (tn *TelegramNotifier) buildTelegramMessage(ctx context.Context, as []*types.Alert) (map[string]string, error) {
	var tmplErr error
	tmpl, _ := TmplText(ctx, tn.tmpl, as, tn.log, &tmplErr)

	msg := map[string]string{}
	msg["chat_id"] = tmpl(tn.ChatID)
	msg["parse_mode"] = "html"

	message := tmpl(tn.Message)
	if tmplErr != nil {
		tn.log.Warn("failed to template Telegram message", "err", tmplErr.Error())
	}

	msg["text"] = message

	return msg, nil
}

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
